templates/singleton/boil_types.go.tpl

```diff
@@ -60,29 +60,81 @@ MySQL output looks like:    TableNameColNameEnumValue = "enumvalue"
 
 It only titlecases the EnumValue portion if it's snake-cased.
 */}}
+
```

Comment:
The whitespace changes in these templates are a bit odd. I notice you're using spaces but these are all tabs (like Go is).